rollup.config.js

```diff
@@ -0,0 +1,252 @@
+import commonjs from 'rollup-plugin-commonjs';
+import babel from 'rollup-plugin-babel';
+import json from 'rollup-plugin-json';
+import autoExternal from 'rollup-plugin-auto-external';
+import cleanup from 'rollup-plugin-cleanup';
```

Comment:
Why would you use this with terser?

Reply:
Also, `prettier` does everything this plugin does. So once/if it's adopted, this plugin would made no sense.

Reply:
The plugin cleanup isn't in usage for the UMD minified bundle where terser is used. I've just added this to the ESM and CJS build pipeline for having a little bit smaller npm packages.

Reply:
Why do we want smaller npm packages? How much of a difference did this plugin made?

Reply:
> Why do we want smaller npm packages? How much of a difference did this plugin made?

It's always better to download less than more if not required. But yes the ``cleanup`` plugin doesn't make a big difference. The more important thing would be to define a better ``npmignore`` or to add a ``files`` property to each package and to remove the ``npmignore`` file. The ``cleanup`` plugin is just a "nice to have" I've added.